File: condicionales/01.cpp
// Dado un número entero, determinar si es positivo, negativo o cero

#include "iostream"
using namespace std;

int main()
{
    int num;
    cout << "Inserte un numero entero: ";
    cin >> num;
    if (num < 0)
    {
        cout << "El numero " << num << " es NEGATIVO";
    }
    else if (num > 0)
    {
        cout << "El numero " << num << " es POSITIVO";
    }
    else
    {
        cout << "El numero " << num << " es CERO";
    }
    return 0;
}

File: condicionales/01.py
num = int(input('Inserte un numero: '))
if num < 0:
    print(f'El numero {num} es NEGATIVO')
elif num > 0:
    print(f'El numero {num} es POSITIVO')
else:
    print(f'El numero {num} es CERO')